index.html

```diff
@@ -30,29 +30,38 @@
     <header>
       <img src="./assets/images/header.png" alt="Doodle Dandy" />
       <nav>
-        <a href="/" class="current">Home</a>
+        <a href="/" class="available">Home</a>
```

Comment:
You need to fix all of the current links from `available` to `current`. Please be careful when doing a global find replace like this.